### ISSUE TITLE
DADA2 version not reported when starting from fasta file

### ISSUE THREAD

**Issue by jtangrot** [nf-core/ampliseq#475]
### Description of the bug

When using ampliseq just for taxonomy assignment, i.e. starting from a fasta file, the DADA2 version used for assignTaxonomy is not displayed in software_versions.yml

### Command used and terminal output

```console
nextflow run nf-core/ampliseq -r dev -profile singularity --input ASV_sequences.fasta --outdir results

This will result in a software_versions.yml containing only the following lines:
CUSTOM_DUMPSOFTWAREVERSIONS:
  python: 3.9.5
  yaml: 5.4.1
QIIME2_INSEQ:
  qiime2: 2021.8.0
Workflow:
  Nextflow: 21.10.6
  nf-core/ampliseq: 2.4.0dev
```

**Comment by d4straub** [comment]
I see, that is indeed too scarce.

**Comment by jtangrot** [comment]
Resolved in #476
